mast/models.py

```diff
@@ -1 +1,69 @@
-# ADD DB CLASSES
\ No newline at end of file
+from mast import db, login_manager
+from flask_login import UserMixin
+import enum
+
+
+@login_manager.user_loader
+def load_user(user_id):
+    return db.session.query(User).get(int(user_id))
+
+
+def get_best_run_activity_by_user(user_id, competition):
+    return db.session.query(Activity).\
+        filter_by(user_id=user_id, type=ActivityType.Run, competition=competition).\
+        order_by(Activity.average_duration_per_km.asc()).\
+        first()
+
+
+class Sex(enum.Enum):
+    Male = 1
+    Female = 2
+
+
+class Age(enum.Enum):
+    Under35 = 1
+    Over35 = 2
+
+
+class ActivityType(enum.Enum):
+    Walk = 1
+    Run = 2
+    Ride = 3
+
+
+class Competition(enum.Enum):
+    Run5km = 1
+    Run10km = 2
+
+
+class User(db.Model, UserMixin):
+    id = db.Column(db.Integer, primary_key=True)
+    username = db.Column(db.String(20), unique=True, nullable=False)
+    name = db.Column(db.String(20), nullable=False)
+    surname = db.Column(db.String(50), nullable=False)
+    email = db.Column(db.String(120), unique=True, nullable=False)
+    password = db.Column(db.String(60), nullable=False)
+    sex = db.Column(db.Enum(Sex))
+    age = db.Column(db.Enum(Age))
+    nick_name = db.Column(db.String(50))
```

Comment:
What is the difference between **_username_** and **_nick_name_**?

Reply:
`username` is for login, `nick_name` (refactored to `display_name`) is used for published results